### PR TITLE
FUSETOOLS-2547 - Explain why keeping Maven Warning

### PR DESCRIPTION
**Pull request by apupier**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -83,6 +83,9 @@
 	</modules>
 
 	<repositories>
+		<!-- Start concerned by comment FUSETOOLS-2547 -->
```
Comment (lhein) [inline pom.xml]: I think concerned by is not the right terminology here. I'd rather put here something like "Start of repository definitions which cause a warning in early build (see FUSETOOLS-2547)"

File: pom.xml [hunk 2]
```diff
@@ -100,6 +103,7 @@
 			<layout>p2</layout>
 			<url>${jbosstools-central-site}</url>
 		</repository>
+		<!-- End concerned by comment FUSETOOLS-2547 -->
```
Comment (lhein) [inline pom.xml]: I'd replace with "End of repository definitions causing a warning (see FUSETOOLS-2547)"

**Comment by apupier** [comment]
testPR

**Comment by apupier** [comment]
testPR

**Pull request merged**